lemur/certificates/cli.py

```diff
@@ -280,46 +289,60 @@ def rotate(endpoint_name, source, new_certificate_name, old_certificate_name, me
             print(
                 f"[+] Rotating endpoint: {endpoint.name} to certificate {new_cert.name}"
             )
+            if not endpoint.primary_certificate:
+                print(f"[!] Endpoint does not have a primary certificate rotate with {new_cert.name}.")
```

Comment:
the print and message lines are redundant, maybe just create a message string and print it then assign it to log_data["message"]